doc/api/examples/map.json

```diff
@@ -5,6 +5,7 @@
         "desc": "Example map for the API",
         "permission": "commons",
         "screenshot": "https://s3.amazonaws.com/metamaps-assets/site/missing-map.png",
+        "starred": false,
```

Comment:
I guess this will be included even if you're signed out? Is there a way to make it behave like 'is_admin' on current_user?